daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -1240,3 +1240,22 @@ class RazerBlade15Advanced2020(_RippleKeyboard):
                'set_ripple_effect', 'set_ripple_effect_random_colour']
 
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1651/razer-blade-15-advanced-2020.png"
+
+class RazerHuntsmanMini(_RippleKeyboard):
+    """
+    Class for the Razer Huntsman Mini
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Huntsman_Mini(-if01)?-event-kbd')
```

Comment:
Ripple not working might be caused by this being wrong
In #1230 these files exist:
```
lrwxrwxrwx 1 root root   9 Aug 18  2020 usb-Razer_Razer_Huntsman_Mini_00000000001A-event-if01 -> ../event8
lrwxrwxrwx 1 root root  10 Aug 18  2020 usb-Razer_Razer_Huntsman_Mini_00000000001A-event-if03 -> ../event13
lrwxrwxrwx 1 root root   9 Aug 18  2020 usb-Razer_Razer_Huntsman_Mini_00000000001A-event-kbd -> ../event6
lrwxrwxrwx 1 root root   9 Aug 18  2020 usb-Razer_Razer_Huntsman_Mini_00000000001A-if01-event-kbd -> ../event7
lrwxrwxrwx 1 root root  10 Aug 18  2020 usb-Razer_Razer_Huntsman_Mini_00000000001A-if02-event-mouse -> ../event12
lrwxrwxrwx 1 root root   9 Aug 18  2020 usb-Razer_Razer_Huntsman_Mini_00000000001A-if02-mouse -> ../mouse1
```
So probably `r'.*Razer_Razer_Huntsman_Mini_00000000001A(-if01)?-event-kbd'` should match it. You can also check the daemon log during startup, that should warn if it can't find any event files